workspaces/README.md

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
oh wait this isn't linked correctly it seems

Reply:
Oh good catch. The newest commit should fix this